liblwgeom/liblwgeom_internal.h

```diff
@@ -122,6 +122,14 @@
 #define SIZE_SET(varsize, size) (((varsize) & 0x00000003)|(((size) & 0x3FFFFFFF) << 2 ))
 
 /**
+* Macro that returns:
+* -1 if n < 0,
+*  1 if n > 0,
+*  0 if n == 0
+*/
+#define SIGNUM(n) ((n > 0) - (n < 0))
```

Comment:
the traditional thing (ha ha, like I'm an expert) for macro defined functions is to put parens around every use of the argument value, so that if the value is itself an expression, the expanded macro won't evaluate in surprising ways, like
```
#define SIGNUM(n) (((n) > 0) - ((n) < 0))
```